meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-bsp/uefi/edk2-firmware-tegra_%.bbappend

```diff
@@ -1,9 +1,4 @@
-
 FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
 
-SRC_URI += "file://0007-runtime-access-for-KernelCommandLine-efivar.patch;patchdir=.."
-SRC_URI += "file://0008-enable-rootfs-redundancy.patch;patchdir=.."
-SRC_URI += "file://0009-BootChainDxe-fix.patch;patchdir=.."
-
-
-
+SRC_URI:append = " file://0007-runtime-access-for-KernelCommandLine-efivar.patch;patchdir=.."
+SRC_URI:append = " file://0009-BootChainDxe-fix.patch;patchdir=.."
```

Comment:
Does anyone know if we still need either or both of these and why exactly they are needed?
I looked back at the history in https://github.com/OE4T/meta-mender-community/pull/18/files and couldn't find these specifically discussed.
I'm wondering whether the BootChainDxe-fix should go upstream.
Assuming these are needed it would be great to add some more detail to the patch header about purpose and include upstream status and signoff header.

Reply:
@dwalkes I tested AGX Xaver and AGX Orin now, both seem to work without this